File: experiments_dataset_vul_detect_cwe_classification/dataset/vul_dataset/java/non-vul/code_before7.java
// Fron cwe-snippets, ./snippets_1/compliant/Java/0176.java

public void checkValid(boolean isValid) {
    if (isValid) {
        System.out.println("Performing processing");
                           doSomethingImportant();
    }
                      else {
        System.out.println("Not Valid, do not perform processing");
                           return;
    }
}

File: experiments_dataset_vul_detect_cwe_classification/dataset/vul_dataset/cpp/non-vul/code_before7.cpp
// From cwe-snippets, snippets_90/non-compliant/C++/cwe-0390/empty_catch_07.cpp

static void func()
{
    char data;
    data = ' ';
    goodB2GSource(data);
    if(data > 0) /* ensure we won't have an underflow */
    {
        if (data < (CHAR_MAX/2))
        {
            char result = data * 2;
            printHexCharLine(result);
        }
        else
        {
            printLine("data value is too large to perform arithmetic safely.");
        }
    }
}